src/application/tasks/ventilation.c

```diff
@@ -23,11 +24,11 @@
  */
 static int start_current_breath_cycle = 0;
 
-/*
- * the time the current breath cycle was supposed to start
- */
-static int breath_cycle_duration = 125; //to do replace this value by the real one
-
+uint32_t calculate_breath_cycle_duration(){
+  uint32_t breath_cycle_duration = 0; // Variable initialization
+  uint32_t respiratory_rate = 15; // Retrieve clinician respiratory rate, THIS IS JUST A PLACEHOLDER FOR NOW

```

Comment:
in your comment add the word todo (it is easier this way to just look into all the files for this keyword)

Reply:
Done